boards/x86_64/qemu_x86_64/qemu_x86_64.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: qemu_x86_64
+name: QEMU Emulation for X86_64
```

Comment:
Not understanding the complaint.  The behavior of an error should be identical to x86 in all ways.  The only difference is that (1) the weak-linked stub is a noop and we do it in the wrapper and (2) if the user-provided override forgets to loop forever, we'll do it on returning.  Is there something you think is going to break?

I mean, the whole point is that our existing architecture is mostly untested and inconsistent.  I agree we want to fix it, I'd just like to avoid being needlessly pedantic about it.

Reply:
Your default policy is to spin forever on any unahndled exception.

The other arches call k_thread_abort() on the offending thread.